twingate/client.go

```diff
@@ -144,22 +141,26 @@ func newServerURL(network, url string) serverURL {
 	return s
 }
 
-func NewClient(url string, apiToken string, network string, version string) *Client {
+func NewClient(url string, apiToken string, network string, httpTimeout time.Duration, httpRetryMax int, version string) *Client {
 	sURL := newServerURL(network, url)
 	retryableClient := retryablehttp.NewClient()
-	retryableClient.HTTPClient.Timeout = Timeout
-	retryableClient.HTTPClient.Transport = newTransport(apiToken, version)
+	retryableClient.RetryMax = httpRetryMax
 	retryableClient.RequestLogHook = func(logger retryablehttp.Logger, req *http.Request, retryNumber int) {
 		log.Printf("[WARN] Failed to call %s (retry %d)", req.URL.String(), retryNumber)
 	}
+	retryableClient.HTTPClient.Timeout = httpTimeout
+	retryableClient.HTTPClient.Transport = newTransport(retryableClient.HTTPClient.Transport, apiToken, version)
+
+	httpClient := retryableClient.StandardClient()
 
 	client := Client{
-		HTTPClient:       retryableClient,
+		RetryableClient:  retryableClient,
+		HTTPClient:       httpClient,
 		ServerURL:        sURL.url,
 		GraphqlServerURL: sURL.newGraphqlServerURL(),
 		APIServerURL:     sURL.newAPIServerURL(),
 		APIToken:         apiToken,
-		GraphqlClient:    graphql.NewClient(sURL.newGraphqlServerURL(), retryableClient.HTTPClient),
+		GraphqlClient:    graphql.NewClient(sURL.newGraphqlServerURL(), httpClient),
```

Comment:
this is the fix, to get a standard HTTP client that supports retries, you need to get an instance from  [retryableClient.StandardClient()](https://github.com/Twingate/terraform-provider-twingate/pull/51/files#diff-134984f9bebbd0ae1f657dc170e63d3c218ac4d72f03ac098fabeb53c7a87183R154)